src/presets/cleavr.ts

```diff
@@ -1,5 +1,6 @@
 import { defineNitroPreset } from '../preset'
 
 export const cleavr = defineNitroPreset({
+  entry: '#internal/nitro/entries/cleavr',
```

Comment:
it doesn't need to have its own entry; by default it will use the node server entry.

I think the issue you encountered was due to a regression in the build pipeline: https://github.com/nuxt/framework/issues/8213#issuecomment-1279786434